app/templates/views/confirm_email_address.html

```diff
@@ -0,0 +1,42 @@
+{% extends "document_download_template.html" %}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{%- from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary -%}
+
+
+{% block per_page_title %}
+  Confirm your email address
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if show_invalid_email_error %}
+        {{ govukErrorSummary({
+          "titleText": "There is a problem",
+          "errorList": [
+            {
+              "text": "Not a valid email address",
+              "href": "#email-address-input"
+            },
+          ]
+        }) }}
+      {% endif %}
+
+      <h1 class="govuk-heading-l">Confirm your email address</h1>
```

Comment:
Seeing as this page only has one question we should use the `isPageHeading` param on the `EmailAddress` form field class to make it output the h1 inside its label. It might be worth putting a Jinja comment in this template to say that's what's happening so future devs aren't wondering where the h1 is.

Reply:
Discussing this offline with @klssmith it seems that the design for this includes the h1 **and** a label for the textbox. That being the case we should stick with this implementation for now and talk to @karlchillmaid about changing it in future.

Reply:
🤔 Fair enough. I'm all for sticking to design system and removing the label in the future 🙌